pkg/login/social/azuread_oauth.go

```diff
@@ -68,10 +68,11 @@ func (s *SocialAzureAD) UserInfo(client *http.Client, token *oauth2.Token) (*Bas
 		return nil, ErrEmailNotFound
 	}
 
-	role, grafanaAdmin := claims.extractRoleAndAdmin(s.autoAssignOrgRole, s.roleAttributeStrict)
-	if role == "" {
+	role, grafanaAdmin := claims.extractRoleAndAdmin()
```

Comment:
nit: I think this is the only method which was using Azure's `autoAssignOrgRole` parameter - it should be fine to remove it altogether now.